docs/source/developer_guide/tutorial.rst

```diff
@@ -14,21 +14,29 @@ The following steps can then be taken, using `ALIGNN-FF <https://github.com/usni
 
 Dependencies for ``janus-core`` are specified through a ``pyproject.toml`` file, with syntax defined by `poetry's dependency specification <https://python-poetry.org/docs/dependency-specification/>`_.
 
-Required dependencies are listed under ``[tool.poetry.dependencies]``, but new MLIPs should initially be added as optional dependencies under ``[tool.poetry.group.extra-mlips.dependencies]``::
+New MLIPs should initially be added as optional dependencies under ``[tool.poetry.dependencies]``, and added as an ``extra`` under ``[tool.poetry.extras]``::
 
-    [tool.poetry.group.extra-mlips]
-    optional = true
-    [tool.poetry.group.extra-mlips.dependencies]
-    alignn = "^2024.5.27"
+    [tool.poetry.dependencies]
+    alignn = { version = "2024.5.27", optional = true }
+    sevenn = { version = "0.9.3", optional = true }
+    torch_geometric = { version = "^2.5.3", optional = true }
+
+    [tool.poetry.extras]
+    alignnff = ["alignn"]
+    sevennet = ["sevenn", "torch_geometric"]
 
 Poetry will automatically resolve dependencies of the MLIP, if present, to ensure consistency with existing dependencies.
 
-These dependencies can then be installed by running:
+.. note::
+    In the case of ``sevennet``, it was necessary to add ``torch_geometric`` as an additional dependency, but in most cases this should not be required
```

Comment:
It might be best to avoid using it as an example if it's a special case then.

Reply:
I mostly added this as well as `alignn` because it's a special case (although partly also since it's useful to show multiple MLIPs for the `--extras` installation).

Most of this tutorial is focused on alignn, but I thought it's probably useful to show how, if the MLIP doesn't come with everything it needs, you'd add that too.

Reply:
Just to add this isn't unique to sevennet, unfortunately. We'd need something similar for m3gnet (matgl) if it was an extra too, so I definitely think an example is useful

Reply:
yes and in theory all the potentials will be extras... and we will have [all] that pulls them all... I am in two minds about making mace as default... I am inclining towards it... but happy to hear options.

Reply:
but this is not about documentations per se...